src/panels/lovelace/cards/hui-humidifier-card.ts

```diff
@@ -84,6 +84,10 @@ export class HuiHumidifierCard extends LitElement implements LovelaceCard {
       `;
     }
 
+    if (!this._setHum) {
+      this._setHum = this._getSetHum(stateObj);
+    }
```

Comment:
This is not the correct fix, we should not have side effects in render.

Can you find out when the humidity is not visible? Is it after a certain action?